app/src/test/java/ch/epfl/sdp/game/GameTest.java

```diff
@@ -122,11 +122,8 @@ public void gameThread_runs() {
     public void gameThreadCatchExceptions() {
         Game.getInstance().initGame();
 
-        Updatable updatable = new Updatable() {
-            @Override
-            public void update() {
-                double a = 1 / 0;
-            }
+        Updatable updatable = () -> {
+            double a = 1 / 0;

```

Comment:
why 1/0? Its undefined?

Reply:
It's to give an exception for test coverage